src/pages/console/auth/import.mdx

```diff
@@ -36,7 +36,7 @@ We can also use Amplify CLI to [import a Cognito user pool resource](/cli/auth/i
 
 ## Import an existing identity pool
 
-![import Amazon Cognito resource](/images/studio/import-identity-pool.png)
+![Enabled option being shown to import an Amazon Cognito identity pool resource](/images/studio/import-identity-pool.png)
```

Comment:
Could we also type out what the option is? Mentioning that it's the "use identity pools" option would help a lot for people using screen readers